app/models/travesty.rb

```diff
@@ -2,8 +2,19 @@ class Travesty < ActiveRecord::Base
   before_save :generate_slug
   has_many :items
 
+  def to_param
```

Comment:
Didnt see this method used anywhere in these files. Does it get called elsewhere in the app?